composer.json

```diff
@@ -1,32 +1,32 @@
 {
-    "name": "kitpages/data-grid-bundle",
+    "name": "imt/kitpages",
     "type": "symfony-bundle",
-    "description": "Symfony DataGridBundle",
+    "description": "IMT DataGridBundle",
```

Comment:
Here I don't understand this "IMT". I'm not sure we have to change the description here.

Reply:
Yes!